c3/parametermap.py

```diff
@@ -66,22 +70,24 @@ def __initialize_parameters(self) -> None:
     def update_parameters(self):
         self.__initialize_parameters()
 
-    def load_values(self, init_point):
+    def load_values(self, init_point, extend_bounds=True):
```

Comment:
@nwittler I am not sure if this `extend_bounds` is the expected good default

Reply:
This default is how it was before - I agree that it's more correct to make it be False but that might break backwards compatibility.